### DISCUSSION TITLE
Hide Index button in the sidebar

### DISCUSSION THREAD

**Discussion by donnekgit** [zim-desktop-wiki/zim-desktop-wiki#2091] [General]
I'm using Zim 0.74.3 on Ubuntu.

Is there any way of hiding the Index button that appears side-on in the sidebar when the side pane is closed (eg using F9)?  I seem to remember that in older versions this button was not there.  

When using the distraction-free editing plugin, it's showing there on the left, so it's a bit of a distraction ...

**Comment by introt** [comment]
There isn't an option for that as far as I'm aware - but given that eg. the left sidebar doesn't show up due to being empty, it should be possible to implement in the future. A cursory glance at a code didn't lead me too far, but for a workaround you may try using eg. [ARandR](https://christian.amsuess.com/tools/arandr/) to create a virtual screen bigger than your physical one to hide it ;)
